PayPalNativePayments/src/test/java/com/paypal/android/paypalnativepayments/PayPalNativeCheckoutClientTest.kt

```diff
@@ -241,7 +241,11 @@ class PayPalNativeCheckoutClientTest {
 
     @Test
     fun `when OnApprove is invoked, onPayPalSuccess is called`() {
-        val approval = mockk<Approval>()
+        val mockOrderID = "mock_order_id"
+        val mockPayerID = "mock_payer_id"
+        val approval = mockk<Approval>(relaxed = true)
```

Comment:
q: is there a way to instantiate an `Approval` object without using mockk?

Reply:
@sshropshire no. Another option is to do `Approval(mockk())`. But it still wont work because i have to mock `Approval` specific behaviour, like `approval.data`. So it has to be a mock